Pinta/MainWindow.cs

```diff
@@ -311,7 +308,7 @@ private void CreateWindow ()
 	}
 	#endregion
 
-	private void CreateMainMenu (WindowShell shell)
```

Comment:
I think these unused parameters should just be removed.
The `WindowShell` being passed in is the same as `this.window_shell` so it's unnecessary.

Reply:
You are right @cameronwhite. We can remove it as a parameter, and the nullability guarantees are the same as before. I've removed that now.